src/bundler.ts

```diff
@@ -221,6 +207,29 @@ export class Bundler {
         return bundleResult;
     }
 
+    private async resolveImport(importData, includePaths) {
+        try {
+            await fs.access(importData.fullPath);
+            importData.found = true;
+        } catch (error) {
+            const underscoredDirname = path.dirname(importData.fullPath);
+            const underscoredBasename = path.basename(importData.fullPath);
+            const underscoredFilePath = path.join(underscoredDirname, `_${underscoredBasename}`);
+            try {
+                await fs.access(underscoredFilePath);
+                importData.fullPath = underscoredFilePath;
+                importData.found = true;
+            } catch (underscoreErr) {
+                for (let i = 0; i < includePaths.length; i++) {
+                    importData.fullPath = path.resolve(includePaths[i], importData.path);
+                    return this.resolveImport(importData, includePaths.slice(i + 1));

```

Comment:
As you're always returning during the first iteration, `for` loop is effectively an `if` statement here.
Therefore, I'd rather we write it as an `if`:
```
// If there are any includePaths
if (includePaths.length != 0) {
    // Resolve fullPath using its first entry
    importData.fullPath = path.resolve(includePaths[0], importData.path);
    // Try resolving import with the remaining includePaths
    const remainingIncludePaths = includePaths.slice(1);
    return this.resolveImport(importData, remainingIncludePaths);
}
```